### PR TITLE
Seat Covers Spacing and Font Adjustments

### PR DESCRIPTION
**Pull request by george-icarcover**
- Sorted colors blk->gr->be with database call
- Modified prices and included "from"
- Double checked image sizing and proportions ( need to confirm for safari mobile)
- Double checked text and padding
- Removed video from seat covers page (will be replaced later)

### CODE REVIEW

File: src/components/PDP/components/WarrantySection.tsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ export default function WarrantySection() {
       break;
   }
 
-  if (pathname === '/seat-covers') {
+  if (pathname.startsWith('/seat-covers') || params.seatType === 'leather') {
```
Comment (devwilliamy) [inline src/components/PDP/components/WarrantySection.tsx:54]: you don't need the second one do you?

File: src/components/hero/dropdown/HomeDropdown.tsx [hunk 2]
```diff
@@ -432,7 +432,7 @@ export default function HomeDropdown({
             id={`mobile-select-${title}`}
             disabled={isDisabled}
             defaultValue={''}
-            value={value}
+            // value={value}
```
Comment (devwilliamy) [inline src/components/hero/dropdown/HomeDropdown.tsx:435]: why do you have to get rid of this?

**Comment by devwilliamy** [comment]
Colors aren't right here, probably missed a page:

https://coverland-2-0-git-ga-seat-covers-spac-362251-coverland-3615f02b.vercel.app/seat-covers


Also not pertaining to this one but should this bnaner be here on all pages?
image

**Comment by devwilliamy** [comment]
I guess if anything

The red text should be bolder
image

More padding on sides
image

Ask Ashley if 'in' should be capitalized cus everything else is almost capitalized
image

I'm not goin to take screenshots of the rest but please make usre the casing matches or double check with Ashley about the casing (ex: Effortlessly clean, from kids to pets -> Effortlessly Clean, From Kids To Pets)
image

Please confirm with Ashley about text, seems like it might be different than what we have for car covers
Current:
image

Figma:
image

## Once again, please take note of casing & double check with Ashley if something isn't consistent

File: src/components/hero/dropdown/HomeDropdown.tsx (hunk 2 above)
Comment (george-icarcover) [reply src/components/hero/dropdown/HomeDropdown.tsx:435]: I was getting a controlled element error, the console was giving error saying I should only one, not both

**Comment by george-icarcover** [comment]
> Colors aren't right here, probably missed a page:
> 
> https://coverland-2-0-git-ga-seat-covers-spac-362251-coverland-3615f02b.vercel.app/seat-covers
> 
> Also not pertaining to this one but should this bnaner be here on all pages? image
> 
> is there anything in particular I should be looking out for for these? let me know
> 
> * Double checked image sizing and proportions ( need to confirm for safari mobile)
> * Double checked text and padding
The banner should be changing pictures and data depending on the searchparams

**Comment by devwilliamy** [comment]
More Padding left of Custom Fit , and right of easy install
image

Caps
image

image

image

image

**Pull request merged**